doc/ref_arch/kubernetes/chapters/chapter06.md

```diff
@@ -10,9 +10,36 @@
 
 <a name="6.2"></a>
-## 6.2 Heading
+##  6.2 Principles
+### Kubernetes Security
+The following are core principles to consider when securing cloud-native applications and infrastructure;
+
+- Deploy only secure applications and trusted codes
+- Only deploy applications from validated and verified images
+- Only deploy applications from trusted registries
+- Containers orchestration (Kubernetes) must be secure with administrative boundaries between tenants
+- Network segmentation using security zoning and network standards must be followed when deploying containers. 
+- The Host OS running the containers must be hardened
+- Use container-aware runtime defense tools.
+
+##  6.3 Kubernetes Security Architecture
+Kubernetes Security architecture is made up of of several components that provide end to end security for the infrastructure and application or workload. These components include image registries (private or public), Kubernetes API, RBAC, network services, host and container runtime, keys or secret management. Each of these components work in a collaborative mode to ensure the security of the applications or workload. 
 
 <a name="6.3"></a>
-## 6.3 Heading
+## 6.4 Security Perimeters
+When applications or workloads run on Kubernetes, there are several layers which come into picture that govern the security. Each of these layers need to be secured within their perimeters. The various layers that come into picture are:
+
+- Container Registry: A container registry is a repository to manage container images. The access to container registry needs to be secured in order to provide a controlled access.
+- Container Images: Containers image is a unit of pre-packages application and all its dependencies. Before loading them to container registry, they need to be made secured by performing various checks like vulnerability analysis, scans etc. These should also be signed and from trusted sources.
+- Containers: A container is a running instance of a container image. The containers need to be prevented from accessing the underlying OS like loading of kernel modules, checking the privilege level of containers.
+- Pods: A pod is a collection of containers.
+- Node: A node can either be a master node (where all the core services are running) or a worker node (where the workloads are running). A node in an unsecured boundary can lead to a potential threat to the running workloads. 
+- Cluster: A cluster comprises of a collection of nodes which contain the control plane as well as the data plane. It also provides API endpoints for interacting with the cluster. The communication over these APIs needs to be secured via different mechanisms like TLS encryption, API authentication via LDAP etc.
```

Comment:
Can we use the definitions in chapter 1 please, rather than re-defining here?

Reply:
Well ! The intent here is not to define containers, pods etc. but to write about the security at various layers (innermost-container to outermost-Cluster). I used the definition so as to get a flow in the description.
Let me know if it addresses your concern, else I'll try to rephrase it.

Reply:
Yes, understood - I don't have a problem with having some definitions here to help with the flow, I just think we should start from the same definitions as are used elsewhere in CNTT (i.e. those in #595).  Maybe provide a link to the Glossary too.

For example, for container image, we could have:
> - Container Images: Stored instance of a container that holds a set of software needed to run an application. Before loading them to container registry, they need to be made secured by performing various checks like vulnerability analysis, scans etc. These should also be signed and from trusted sources.

The first sentence is the definition from the glossary, and the rest is the wording you've added to provide the context/flow.

Reply:
Ok, sure. I'll update it.

Reply:
Fixed.